SU-HKKU/cardinal_service_profile
# CSD-269 improve csv Importer

# READY FOR REVIEW

# Summary
- Use migrate module for CSV importer instead of the contrib module that provides little flexibility.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout the stack branch and do `composer install -n`
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-269_improve-csv`
1. `blt drupal:sync --site=cardinalservice -n`
1. go to `/admin/content/import` page after logging in
1. download one of the templates from the link at the top of the page
1. populate it with some various rows
1. remove the helper text in the header row. removing the words in the parentheses including the parentheses.
1. upload it and run the import.
1. verify your content was created `/admin/content`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
